### PR TITLE
All German translations in strings have been reviewed

### PR DESCRIPTION
**Pull request by patepelo**
The German translation has been reviewed, and many terms have been asked to the DACH (German speaking) community. 

A brief review from another German speaker is requested and appreciated.

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -6440,7 +6440,7 @@
     ca = Notícies
     cs = Novinky
     da = Nyheder
-    de = News
+    de = Neuigkeiten
```
Comment (Misalf-git) [inline data/strings/strings.txt:6443]: In the past, the original german term was deliberateley changed to the english, yet common term "News".

File: data/strings/strings.txt [hunk 2]
```diff
@@ -12252,7 +12252,7 @@
     ca = Descriviu el problema detalladament perquè la comunitat d'OpenStreetMap pugui corregir l'error.
     cs = Popište prosím detailně problém, aby vám komunita OpenStreetMap mohla pomoct opravit chybu.
     da = Beskriv problemet i detaljer, så OpenStreetMap-fællesskabet kan løse fejlen.
-    de = Bitte beschreiben Sie das Problem detailliert, damit die OpenStreetMap-Gemeinschaft den Fehler beheben kann.
+    de = Bitte beschreiben Sie das Problem detailliert, damit die OpenStreetMap-COmmunity den Fehler korrigieren kann.
```
Comment (Misalf-git) [inline data/strings/strings.txt:12255]: Community

File: data/strings/strings.txt [hunk 3]
```diff
@@ -14780,7 +14780,7 @@
     ca = Error en iniciar sessió.
     cs = Chyba při přihlašování.
     da = Login-fejl.
-    de = Login-Fehler.
+    de = Anmeldungsfehler.
```
[the remark this answers is not in the document]
Comment (Misalf-git) [reply data/strings/strings.txt:14783]: I'd prefer "Login-Fehler".
Translation shouldn't just convert every word into the other language, but make it understandable. Several german words don't bring benefit from being translated from english.

File: data/strings/strings.txt [hunk 4]
```diff
@@ -16046,7 +16046,7 @@
     ca = Envia una ota als editors d'OSM
     cs = Odeslat poznámku editorům OSM
     da = Send en besked til OSM-redaktører
-    de = Notiz an OSM-Redakteure senden
+    de = Notiz an OSM-Beitragender senden
```
[the remark this answers is not in the document]
Comment (Misalf-git) [reply data/strings/strings.txt:16049]: "Notiz an Freiwillige *von* OpenStreetMap senden"

_Die OpenStreetMap_ wäre verdeutschung von _der (**die**) Map_ (sächlich), gemeint ist aber die Vereinigung.

File: data/strings/strings.txt [hunk 5]
```diff
@@ -16397,7 +16397,7 @@
     ca = Es desconeix la ubicació actual.
     cs = Současná poloha nezjištěna.
     da = Aktuel position er ukendt.
-    de = Standort nicht gefunden.
+    de = Aktueller Standort ist unbekannt
```
Comment (Misalf-git) [inline data/strings/strings.txt:16400]: "Aktueller Standort unbekannt"

File: data/strings/strings.txt [hunk 6]
```diff
@@ -18470,7 +18470,7 @@
     ca = Revisarem els canvis. Si tenim cap pregunta contactem amb vós via correu electrònic.
     cs = Redaktoři OpenStreetMap zkontrolují změny a budou vás kontaktovat, pokud budou mít nějaké dotazy.
     da = OpenStreetMap-redaktører vil tjekke ændringerne og kontakte dig, hvis de har spørgsmål.
-    de = OpenStreetMap-Redakteure werden die Änderungen prüfen und sich bei Fragen mit Ihnen in Verbindung setzen.
+    de = OpenStreetMap-Beitragender werden die Änderungen prüfen und sich bei Fragen mit Ihnen in Verbindung setzen.
```
[the remark this answers is not in the document]
Comment (Misalf-git) [reply data/strings/strings.txt:18473]: "Freiwillige _von_ ..."

File: data/strings/strings.txt [hunk 7]
```diff
@@ -24030,7 +24030,7 @@
     ca = Definiu les vies a evitar
     cs = Definovat silnice, kterým se vyhnout
     da = Definer veje du vil undgå
-    de = Umwege einstellen
+    de = Definieren Sie zu vermeidende Straßen
```
Comment (Misalf-git) [inline data/strings/strings.txt:24033]: "Routenbeschränkung einstellen"

This switched back to "Umwege" several times for some reason...

**Comment by Misalf-git** [comment]
Added some mustard as well. I hope it worked this time.

File: data/strings/strings.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (Misalf-git) [reply data/strings/strings.txt:16049]: Jo, oder "ihr seine ...", hauptsache neutral.

File: data/strings/strings.txt (hunk 1 above)
Comment (patepelo) [reply data/strings/strings.txt:6443]: Then I'll let it be as "News"

File: data/strings/strings.txt (hunk 7 above)
Comment (patepelo) [reply data/strings/strings.txt:24033]: OK. Changed to "Routenbeschränkung einstellen". Please let me know if you'd prefer Umwege. Maybe is shorter but it's not a precise translation.

File: data/strings/strings.txt [hunk 8]
```diff
@@ -15520,7 +15520,7 @@
     ca = Inicieu sessió perquè altres usuaris puguin veure els canvis que heu fet
     cs = Přihlaste se, aby ostatní uživatelé mohli vidět změny, které jste provedli.
     da = Log på så andre brugere kan se ændringerne som du har foretaget.
-    de = Melden Sie sich an, damit andere Benutzer Ihre Änderungen sehen können.
+    de = Melden Sie sich bei openstreetmap.org an, um Ihre Änderungen die Welt zu veröffentlichen.
```
[the remark this answers is not in the document]
Comment (patepelo) [reply data/strings/strings.txt:15523]: "Teilen" is more like sharing and could be confusing. It must be clear that changes are published. I will keep your first suggestion for that reason if that's ok.

**Comment by patepelo** [comment]
> Added my mustard as well. I hope it worked this time.

Vielen Dank, @Misalf-git und @j13m126 🙏 

It's now implemented and ready to merge

File: data/strings/strings.txt [hunk 9]
```diff
@@ -13200,7 +13200,7 @@
     ca = Registre a OpenStreetMap
     cs = Registrace
     da = Tilmeld dig
-    de = Registrieren
+    de = Bei OpenStretMap registrieren
```
[the remark this answers is not in the document]
Comment (patepelo) [reply data/strings/strings.txt:13203]: Changed. Thanks.

**Review by patepelo — COMMENTED** [review]
Last typo edited

**Review by vng — APPROVED** [review]
Please, rebase, squash and DCO

**Comment by patepelo** [comment]
> @patepelo could you finish it please? or do you need help?

Yes, give me until tomorrow to try on my own with all the git commands and if I get stuck again, I'll contact you for help 👍

**Comment by pastk** [comment]
Ready to merge!

**Comment by patepelo** [comment]
> Ready to merge!

Many many thanks, @pastk 🙏

**Review by biodranik — APPROVED** [review]
Ich danke Ihnen allen sehr!

**Pull request merged**